pipeline/common/vcf_files_helper.py

```diff
@@ -0,0 +1,40 @@
+from .hgvs_utils import HgvsWrapper
+from .variant_utils import VCFVariant
+import hgvs
+from seq_utils import SeqRepoWrapper
+
+
+def cdna_str_to_genomic_var(cdna_hgvs_str, assembly=HgvsWrapper.GRCh38_Assem, hgvs_wrapper = HgvsWrapper.get_instance(),
+                            seq_fetcher=SeqRepoWrapper.get_instance()):
+    """
+    Convert a CDNA HGVS string into a genomic coordinate.
+    Normalizes in genomic space.
+    Note, that the seq_fetcher needs to be in the same assembly as given by the `assembly` argument.
```

Comment:
is there a way to enforce a check on this? e.g. assert statement

Reply:
yes, good point! Fixed in 1a5201dc7524c5f7459048dd14d8b9d24eea32d4